src/components/Activities/Activity3/Game/FinalReport.tsx

```diff
@@ -81,8 +87,13 @@ function FinalReport(props: FinalReportProps): JSX.Element {
               <div className='bar' style={{ background: getGradient(timePercentages) }}>
                 {Object.values(timeAllocation).map((t, i) =>
                   <>
-                    <div data-tip data-for={`time-${i}`} key={i} style={{ width: `${timePercentages[i]}%`, height: '30px' }}></div>
-                    <ReactToolTip id={`time-${i}`}>{t} days</ReactToolTip>
+                    {timePercentages[i] > PERCENT_THRESHOLD.DAYS ?  //percentage where the text no longer fits inside the corresponding section, note this is not a pixel value, and rather a percentage based on my laptop screen
```

Comment:
nit: i dont think we need this here since its part of the info for `PERCENT_THRESHOLD` rather than just inlining it random code that uses it. if we do the change as per below, then the explanation will appear on hover
```suggestion
                    {timePercentages[i] > PERCENT_THRESHOLD.DAYS ? 
```